tiatoolbox/wsicore/wsireader.py

```diff
@@ -136,11 +136,14 @@ def is_ngff(path: pathlib.Path, min_version: Tuple[int, ...] = (0, 4)) -> bool:
         tuple(int(part) for part in scale.get("version", "").split("."))
         for scale in multiscales
     )
-    omero_version = tuple(int(part) for part in omero.get("version", "").split("."))
+    omero_version = omero.get("verion")
+    if omero_version:
```

Comment:
shouldn't it log a warning if this is empty?

Reply:
It's perfectly valid to be empty according to the spec